### PR TITLE
TDR-2110 - File Level add new status property ClientChecksumStatus

### PR DESCRIPTION
**Pull request by vimleshtna**

### CODE REVIEW

File: src/test/scala/uk/gov/nationalarchives/tdr/api/service/FileServiceSpec.scala [hunk 1]
```diff
@@ -401,11 +406,22 @@ class FileServiceSpec extends AnyFlatSpec with MockitoSugar with Matchers with S
         case _ => count should equal(2)
       }
     })
+    verify(fileStatusRepositoryMock, times(2)).addFileStatuses(any[List[FilestatusRow]]())
```
Comment (MancunianSam) [inline src/test/scala/uk/gov/nationalarchives/tdr/api/service/FileServiceSpec.scala:409]: This is a duplicate of the verify call on line 379. You can get rid of one of them.

File: src/test/scala/uk/gov/nationalarchives/tdr/api/service/FileServiceSpec.scala [hunk 2]
```diff
@@ -461,6 +479,20 @@ class FileServiceSpec extends AnyFlatSpec with MockitoSugar with Matchers with S
...
```
Comment (MancunianSam) [inline src/test/scala/uk/gov/nationalarchives/tdr/api/service/FileServiceSpec.scala:482]: You've got two calls to `verify` in this test as well, you can delete one.

**Review by MancunianSam — COMMENTED** [review]
Looks good thanks. There's just a couple of duplicate verify calls in your tests.

File: src/test/scala/uk/gov/nationalarchives/tdr/api/service/FileServiceSpec.scala (hunk 1 above)
Comment (vimleshtna) [reply src/test/scala/uk/gov/nationalarchives/tdr/api/service/FileServiceSpec.scala]: Removed. Thanks for highlighting it, probably it's a copy-paste error

**Review by MancunianSam — APPROVED** [review]

**Pull request merged**